tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -115,61 +122,57 @@ kbs_set_resource_from_file() {
 	path+="${type}/"
 	path+="${tag}"
 
-	kbs-client --url "$(kbs_k8s_svc_http_addr)" config \
-		--auth-private-key "$KBS_PRIVATE_KEY" set-resource \
-		--path "$path" --resource-file "$file"
+	docker run \
+		-v "$path:$path" \
+		-v "$file:$file" \
+		--privileged \
+		--net=host \
+			ghcr.io/kata-containers/kata-containers/coco-kbs-client:${cli_features}-${rust_version}-${trustee_version}-${uname -m} -- \
```

Comment:
Can we use `$(kbs_get_cli_container_image_name)` here, so that if we change the name of the container we only have to do it in one place?

Reply:
Done!